Clicker/ViewControllers/EditPollViewController.swift

```diff
@@ -125,12 +157,24 @@ class EditPollViewController: UIViewController {
                 make.centerY.equalToSuperview()
                 make.width.equalToSuperview().multipliedBy(editNameButtonWidthScaleFactor)
             }
+        } else {
+            reopenButton.snp.makeConstraints { make in
+                make.leading.equalToSuperview().offset(reopenButtonLeadingPadding)
+                make.centerY.equalToSuperview()
+                make.width.equalToSuperview().multipliedBy(reopenButtonWidthScaleFactor)
+            }
+
+//            reopenView.snp.makeConstraints { make in
```

Comment:
is this being used?

Reply:
not right now b/c the functionality for reopening is unclear - i can delete it for now